client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +49,28 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    private final ReaderGroupRetentionConfig retentionConfig;
+
+    public enum ReaderGroupRetentionConfig {
```

Comment:
this config also seems to be part of anirudh's PR. can we ensure that we consolidate it.

Reply:
Sure. Once his PR is merged I will remove this part and adjust my code to work with his changes.